module/behaviour/skills/KickScript/src/KickScript.cpp

```diff
@@ -103,23 +98,13 @@ namespace module::behaviour::skills {
                            {std::pair<float, std::set<LimbID>>(
                                0,
                                {LimbID::LEFT_LEG, LimbID::RIGHT_LEG, LimbID::LEFT_ARM, LimbID::RIGHT_ARM})},
-                           [this](const std::set<LimbID>&) { emit(std::make_unique<ExecuteKick>()); },
-                           [this](const std::set<LimbID>&) { emit(std::make_unique<FinishKick>()); },
-                           [this](const std::set<ServoID>&) { emit(std::make_unique<FinishKick>()); }}));
+                           [this](const std::set<LimbID>& /*unused*/) { emit(std::make_unique<ExecuteKick>()); },
```

Comment:
If you can, rather than putting /*unused*/ put an actual name for the variables (still commented out)

Reply:
I wasn't especially sure what to put for these. I have gone with fairly obvious names that don't say much.